applications/api-cat/src/main/java/no/acat/service/ElasticsearchService.java

```diff
@@ -63,7 +70,7 @@ public Client getClient() {
         return elasticsearch == null ? null : elasticsearch.getClient();
     }
 
-    private void initializeElasticsearchTransportClient() {
+     void initializeElasticsearchTransportClient() {
```

Comment:
Jeg anbefaler å ikke fortsette med unit-testing av dette service, fordi vi planlegger å endre dette løsning i stor del.

David jobber akkurat nå med løsning av auto-reindex.  Se på dette bransje: feature/ds/auto-alias-mapping-update

Vi har to løsninger til å koble oss til elasticsearch, og vi må løse auto-reindex issue så snart så mulig på begge løsninger. Etter vi er ferdig med dette, skal vi unifisere til bare ett løsning - det skal bli spring data elastic repository basert, og det mener at helle denne elasticsearch tjeneste blir slettet.